SnooksHarvard.csl

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title> Snooks Australia Harvard (author-date)</title>
+    <id>http://www.zotero.org/styles/SnooksHarvard</id>
+    <link href="/http[s]?:\/\/www.zotero.org\/styles\/SnooksHarvard/" rel="self"/>
+    <link href="http://www.australia.gov.au/publications/style-manual" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/harvard1" rel="template"/>
+    <author>
+      <name>jstedman</name>
+      <email>jessica.stedman@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Snooks style manual Australia Harvard author-date style</summary>
+    <updated>2014-09-10T06:18:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+         <terms>
+	<term name="edition" form="short">edn</term>
+</terms>  
+</locale>
+ <macro name="editor"> 
+    <names variable="editor" delimiter=" " prefix="">
+      <name and="symbol" initialize-with="" delimiter=", "/>
+      <label form="short" prefix=" (" suffix=")" text-case="lowercase"/>
+    </names>
+  </macro>
+  <macro name="anon">
```

Comment:
this is confusing. If items with no author/editor should just print the title, put that into the author macros directly. (i.e. use ```<text macro="title"/>``` there instead of the anon macro. Then delete this one.